dox/books/Bill_Primer/DoxyLangman

```diff
@@ -788,10 +788,7 @@ RECURSIVE              = NO
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = ./test_* \
-                         ~/.. ./.. .. ../.. ../../.. \
-                         /home/ /home/j/ /home/j/dev \
-                         dox/books/Bill_Primer/Bill_Primer.dox
+EXCLUDE                = dox/books/Bill_Primer/Bill_Primer.dox
```

Comment:
I am confused.  Why does language manual have INPUT set to Bill_Primer and exclude Bill_Primer.dox?  Is this a placeholder until you start this book?